src/controllers/subject.js

```
@@ -15,6 +15,20 @@ const getSubjects = async (req, res) => {
   res.status(200).json(subjects);
 };
 
+const createSubject = async (req, res) => {
+  const subjectData = req.body;
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)